build_docs.pl

```diff
@@ -352,18 +352,23 @@ sub check_kibana_links {
 # ${baseUrl}guide/en/elasticsearch/reference/${urlVersion}/modules-scripting-expression.html
 # ${ELASTIC_WEBSITE_URL}guide/en/beats/filebeat/${DOC_LINK_VERSION}
 # ${ELASTIC_DOCS}search-aggregations-bucket-datehistogram-aggregation.html
+# ${ELASTICSEARCH_DOCS}update-transform.html
+# ${KIBANA_DOCS}canvas.html`
+# ${PLUGIN_DOCS}repository-s3.html
 
     my $extractor = sub {
         my $contents = shift;
         return sub {
-            while ( $contents =~ m!`(\$\{(?:baseUrl|ELASTIC.+)\}[^`]+)`!g ) {
+            while ( $contents =~ m!`(\$\{(?:baseUrl|ELASTIC.+|KIBANA.+|PLUGIN.+)\}[^`]+)`!g ) {
```

Comment:
```suggestion
            while ( $contents =~ m!`(\$\{(?:baseUrl|ELASTIC.+|KIBANA_DOCS|PLUGIN_DOCS)\}[^`]+)`!g ) {
```
What you have is fine, but the `ELASTIC.+` was meant to catch all of `ELASTIC_DOCS`, `ELASTICSEARCH_DOCS`, and `ELASTIC_WEBSITE_URL`.